computer_vision_draft/houghTransform/houghTransform/main.cpp

```diff
@@ -0,0 +1,106 @@
+#include <opencv2/opencv.hpp>
+#include <iostream>
+#include <string>
+#include <fstream>
+#include<windows.h>  
```

Comment:
Should probably keep indentation consistent if we are making any changes anyways, not needed on its own but if any other changes are made we might as well.

Reply:
Applied the suggestion